public/stylesheets/main.css

```
@@ -13,14 +13,27 @@ body { padding-top: 70px; }
     border-radius: 0 0 0.5em 0.5em;
 }
 
-#packer-output {
-    max-height: 400px; /* This gets updated to match the screen size using JS */
-    overflow-y: scroll;
+.well {
     background-color: #15191F; /* iTerm2's default theme */
+    overflow-y: scroll;
+    max-height: 400px; /* This gets updated to match the screen size using JS */
+    color: #DCDCDC; /* iTerm2's default theme */
 }
 
-.bake-log {
-    color: #DCDCDC; /* iTerm2's default theme */
+.package-diff-insert {
+    color:rgb(239, 246, 252);
+    background: rgb(10, 108, 54);
```

```suggestion
    color: #EFF6FC;
    background: #0A6C36;
```

A small change to keep consistent colour declaration styles within the repo.